ui/pages/[page].js

```diff
@@ -17,9 +17,18 @@ const StaticPage = ({
   return (
     <Page title={title} description={description} host={host}>
       <div className="nhsuk-grid-row">
-        {showToc && <TableOfContents content={parsed} />}
+        <div className="nhsuk-grid-column-one-third">
+          <br />
+        </div>
```

Comment:
😬

Reply:
To clarify the outcome is good. It's just a shame to need such a wild hack to get there.

Reply:
indeed